iphone/Classes/TiContactsPerson.m

```diff
@@ -605,16 +597,16 @@ -(TiBlob*)image
 -(void)setBirthday:(NSString*)date
 {
 	ENSURE_UI_THREAD(setBirthday, date)
-#if IS_XCODE_7
-	if ([TiUtils isIOS9OrGreater]) {
+
+    if ([TiUtils isIOS9OrGreater]) {
 		NSDate *saveDate = [TiUtils dateForUTCDate:date];
```

Comment:
Indentation , all the indentation looks good in code but messed up on github. Think some its spaces and you are using tabs or the other way around